src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy

```diff
@@ -28,16 +28,11 @@ class BugsnagVariantOutputTask extends DefaultTask {
         // when splits are enabled, the output has a name with the following structure:
         // e.g. "javaExample-hdpiMips-debug"
 
-        if (tokens.length == 3) {
-            def split = tokens[1]
-
-            if (SPLIT_UNIVERSAL == split) {
-                directory = new File(directory, SPLIT_UNIVERSAL)
-            } else {
-                def density = findValueForDensityFilter(split, project.ext.splitsInfo.densityFilters)
-                def abi = findValueForAbiFilter(split, project.ext.splitsInfo.abiFilters)
-                directory = findManifestDirForSplit(density, abi, directory)
-            }
+        if (tokens.length == 4) { // should be one of the following cases: 'armeabi-v7a', 'arm64-v8a'
+            def split = tokens[1] + "-" + tokens[2] // combine back into one string
+            directory = guessManifestDir(directory, split)
+        } else if (tokens.length == 3) {
+            directory = guessManifestDir(directory, tokens[1])
```

Comment:
Should probably have another case here for if there are more than 4 tokens to throw a meaningful error and print some debugging info

Reply:
thanks @kattrali - I've added in logging if there are more than 4 tokens.